docs/_static/css/custom.css

```diff
@@ -2,4 +2,7 @@
 
 .wy-nav-content {
     max-width: none;
-}
\ No newline at end of file
+}
+.wy-table-responsive table td, .wy-table-responsive table th {
+    white-space: inherit;
+}
```

Comment:
set table width dynamically to fit to screen width. See e.g.: the Components docs: https://power-grid-model--409.org.readthedocs.build/en/409/user_manual/components.html